### ISSUE TITLE
each-in doesn't support a second argument

### ISSUE THREAD

**Issue by boris-petrov** [lifeart/els-addon-typed-templates#48]
```handlebars
{{#each-in this.hash as |key value|}}
  {{yield value}}
{{/each-in}}
```

Leads to:

```
Tuple type '[unknown]' of length '1' has no element at index '1'.typed-templates
```

**Comment by lifeart** [comment]
Hi @boris-petrov! Thank you for report! This types located in https://github.com/lifeart/els-addon-typed-templates/blob/master/src/lib/common-types.d.ts#L91

I will be able to update code in a day or such, also, any typing updates are welcome

**Comment by lifeart** [comment]
fixed in v4.0.4
